fish_right_prompt.fish

```diff
@@ -0,0 +1,17 @@
+function fish_right_prompt
+
+	# ------------------------------------------------------------------------------
+	# Configuration
+	# ------------------------------------------------------------------------------
+
+	__sf_util_set_default SPACEFISH_RPROMPT_ORDER ""
+
+	# ------------------------------------------------------------------------------
+	# Sections
+	# ------------------------------------------------------------------------------
+
```

Comment:
I would suggest adding an early return here if `SPACEFISH_RPROMPT_ORDER` remains empty. This should do the trick:
```fish
[ -n "$SPACEFISH_PROMPT_ORDER" ]; or return
```